src/cli.rb

```diff
@@ -43,6 +43,8 @@ def run
       program :version, '2017.2.1'
       program :description, 'Alces tools for the management and configuration of bare metal machines'
 
+      suppress_trace_class UserMetalwareError
```

Comment:
A comment might be helpful here to indicate that this new option comes from our fork, and what this does.

Reply:
Added